iOS-Email-Client/Managers/DBManager.swift

```diff
@@ -327,10 +327,11 @@ class DBManager: SharedDB {
             }
             thread.unread = threadEmails.contains(where: {$0.unread})
             thread.counter = threadEmails.count
-            thread.subject = threadEmails.first!.subject
+            thread.subject = threadEmails.last!.subject
```

Comment:
We still want to show the original subject of the thread, avoiding `Re:` & `Fw:` prefixes